csrankings-c.csv

```diff
@@ -1149,6 +1149,7 @@ Cihan Tunc,University of North Texas,https://computerscience.engineering.unt.edu
 Cihang Xie,Univ. of California - Santa Cruz,https://cihangxie.github.io,X3vVZPcAAAAJ
 Cindy Grimm,Oregon State University,http://web.engr.oregonstate.edu/~grimmc,pdCI7sAAAAJ
 Cindy L. Bethel,Mississippi State University,http://www.cavs.msstate.edu/directory/information.php?d=2258,Alsv0h8AAAAJ
+Cindy Kaiying Lin,Pennsylvania State University,https://ist.psu.edu/directory/cfl5598,stSffQ8AAAAJ

```

Comment:
Not in EECS. https://www.eecs.psu.edu/departments/cse-faculty-list.aspx

Reply:
Please see my previous comment. Cindy works in HCI (an ethnographer of computing infrastructures and environmental governance) and publishes in ACM CHI.